src/hotspot/share/gc/g1/g1Analytics.cpp

```diff
@@ -150,11 +151,19 @@ void G1Analytics::report_alloc_rate_ms(double alloc_rate) {
 }
 
 void G1Analytics::compute_pause_time_ratios(double end_time_sec, double pause_time_ms) {
+  // consider the most recent Remark and/or Cleanup pause time.
+  // look at append_prev_collection_pause_end_ms() for clues on why we use
+  // _prev_collection_pause_end_ms to determine the pause time
```

Comment:
Upper-case at start of sentence and is it possible to be more explicit than handing out clues :)